domains/certificates/Issuer/Issuer.Worker/Program.cs

```diff
@@ -5,35 +5,34 @@
 using Issuer.Worker.MasterDataService;
 using Issuer.Worker.QueryModelUpdater;
 using Issuer.Worker.RegistryConnector;
+using Issuer.Worker.RegistryConnector.Health;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Serilog;
 using Serilog.Formatting.Json;
 
-var host = Host.CreateDefaultBuilder(args)
-    .ConfigureLogging((context, builder) =>
-    {
-        builder.ClearProviders();
+var builder = WebApplication.CreateBuilder(args);
+var logger = new LoggerConfiguration()
+    .WriteTo.Console(new JsonFormatter())
```

Comment:
Something has been lost here: Before when environment is "Development", then it did `WriteTo.Console()` and when not in "Development" it did `WriteTo.Console(new JsonFormatter())`. It is nice to have an easier to read console during development.